web-admin/src/features/projects/github/ProjectGithubConnection.svelte

```diff
@@ -58,6 +65,11 @@
       },
     );
   }
+
+  function disconnectGithubConnect() {
+    void githubData.promptGithubLogin();
```

Comment:
Based on the behavior, given the user isn't always shown a GitHub Login page, it feels like this function name should read something like "ensureLoggedInToGitHub".

Reply:
`ensureGithubAccess` makes more sense since we are checking and using `grantAccessUrl`